docs/guides/api-gateway/kubernetes.mdx

```diff
@@ -0,0 +1,414 @@
+---
+description: Deploy ngrok's Kubernetes Operator onto your cluster for unified ingress and advanced traffic shaping with K8s-native interfaces.
+---
+
+import TabItem from "@theme/TabItem";
+import Tabs from "@theme/Tabs";
+
+# Deliver and secure ingress for APIs in Kubernetes
+
+You currently have a K8s cluster with an API (micro)service and want to deliver it
+to production with additional features and protection, like rate limiting,
+authentication, and DDoS protection.
+
+In this guide, you'll learn how to implement ngrok as an API gateway, using the
+ngrok Kubernetes Operator, in these steps:
+
+1. [Apply the ngrok Kubernetes Operator to your cluster](#apply-kubernetes-operator)
+2. [Configure your cluster to use ngrok as an API gateway](#configure-cluster-api-gateway)
+3. [Set up Traffic Policy to shape traffic into/out of your API service](#set-up-traffic-policy)
+
+## What you'll need
+
+<div className="ngrok--cards ngrok--cards-row">
+	<div className="ngrok--card ngrok--card-sm">
+		<div className="ngrok--card-heading jc-space-between">
+			<h3 className="fw-600">ngrok account</h3>
+		</div>
+		<p>
+			Any tier will do&mdash;
+			<a href="https://dashboard.ngrok.com/signup">register now</a>.
+		</p>
+	</div>
+	<div className="ngrok--card ngrok--card-sm">
+		<div className="ngrok--card-heading jc-space-between">
+			<h3 className="fw-600">A reserved domain</h3>
+		</div>
+		<p>
+			Grab yours in the{" "}
+			<a href="https://dashboard.ngrok.com/">ngrok dashboard</a>.
+		</p>
+	</div>
+</div>
+<div className="ngrok--cards ngrok--cards-row">
+	<div className="ngrok--card ngrok--card-sm">
+		<div className="ngrok--card-heading jc-space-between">
+			<h3 className="fw-600">A Kubernetes cluster</h3>
+		</div>
+		<p>
+			... and optionally, a functioning API service. If you don't yet have one,
+			you can install our demo service.{" "}
```

Comment:
you can install our [demo service](https://github.com/ngrok-samples/ngrok-api-service).{" "}

Reply:
Add link to GitHub.